refinery/ui/source/js/dashboard/partials/permission-dialog.html

```diff
@@ -2,8 +2,10 @@
   <h3>Permission Editor</h3>
   <div class="modal-description">
     <h4>
-      Select which groups you are a member of should have access
-      to this data set. Assign either Read-only or Modify permissions.
+      Select groups you would like to access this data set.
+      To share this data set with a new group or add a user
+      to a group, go to <a href=/collaboration> the collaboration page.</a>
```

Comment:
While unquoted attribute values are valid in HTML5 should we standardize on double-quotes for uniformity and just to be on the safe side?
https://www.w3.org/TR/2012/WD-html-markup-20120320/syntax.html#attr-value-unquoted